src/commands/phpmyadmin.ts

```diff
@@ -159,15 +161,43 @@ export class PhpMyAdminCommand {
 		return await getPhpMyAdminStatus( this.app.id as number, this.env.id as number );
 	}
 
+	async readyToServe(): Promise< boolean > {
+		const url = `https://${ this.env.primaryDomain?.name }/.wpvip/pma/health`;
+
+		const agent = createProxyAgent( url );
+		const resp = await fetch( url, {
+			method: 'GET',
+			redirect: 'manual',
+			agent: agent ?? undefined,
+		} );
```

Comment:
Should we add some cache-ignoring/cache-busting headers/cookies here? Or switch to `POST` here? It will at least eliminate the possibility that it's not the cache that's causing issues.